dietpi/dietpi-drive_manager

```diff
@@ -1369,7 +1369,7 @@ aDRIVE_ISPARTITIONTABLE ${aDRIVE_ISPARTITIONTABLE[$i]}
 					G_WHIP_YESNO "Your user data will be moved:\n - From: $FP_USERDATA_CURRENT\n - To: $fp_target_userdata_dir\n\nDo you wish to continue?"
 					if (( ! $? )); then
 
-						/DietPi/dietpi/func/dietpi-set_userdata "$FP_USERDATA_CURRENT" "$fp_target_userdata_dir"
+						SKIP_SERVICES=1 /DietPi/dietpi/func/dietpi-set_userdata "$FP_USERDATA_CURRENT" "$fp_target_userdata_dir"
```

Comment:
I was thinking of having a global for this, which could be used in any situation, eg:
```
export G_DIETPI-SERVICES_DISABLE=1
/DietPi/dietpi/func/dietpi-set_userdata "$FP_USERDATA_CURRENT" "$fp_target_userdata_dir"
export G_DIETPI-SERVICES_DISABLE=0
```

I will probably merge and make the change in another commit

Reply:
@Fourdee 
Jep same idea here 👍. Would bring consistency inside and also sub-sub-scripts then directly know if they need to handle or not.